src/VisualStudio/CSharp/Repl/CSharpInteractiveCommandHandler.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.CSharp.Interactive
 {
-    [ExportCommandHandler("Interactive Command Handler", ContentTypeNames.CSharpContentType)]
+    [ExportCommandHandler("Interactive Command Handler")]
```

Comment:
Does Execute in interactive and Copy to interactive still work in a .cs file with this?